Arduino/Ground Systems/Controls/controls.ino

```diff
@@ -1,19 +1,21 @@
-#include <MCP23S17.h>
+#include <MCP23S17.h> // MCP23S17 by Rob Tillaart (https://github.com/RobTillaart/MCP23S17)
 
 #define NVALVES 16
 #define BUFFER 1000
 
-MCP23S17 io(&SPI, 10, 0);
+MCP23S17 io(10, 0, &SPI);
```

Comment:
What does this change?

Reply:
The constructor provided by the MCP23S17 library used in this updated implementation is invoked in a different way than what the original source code did. I assume a different library was used in the past with a slightly different API.